blues/formats.py

```diff
@@ -121,8 +167,13 @@ def write(self, coordinates, parameters=None, environment=None,
             You may optionally specify the temperature in each frame. By
             convention the temperatures should b in units of Kelvin.
         alchemicalLambda : np.ndarray, shape=(n_frames,), optional
-            You may optionally specify the alchemical lambda in each frame. These
+            You may optionally specify the alchemicalLambda in each frame. These
             have no units, but are generally between zero and one.
+        protocolWork : np.ndarray, shape=(n_frames,), optional
+            You may optionally specify the protocolWork in each frame. These
```

Comment:
Um... Would you want to specify the protocolWork? Seems like it's something that is accumulated automatically based on your protocol and specifying it would usually cause problems?

Reply:
This is to let you store the protocolWork at the starting frame, midpoint, and end point along with the coordinates for visualization.